lab/src/raft/append_entries.go

```diff
@@ -40,7 +40,9 @@ func (rf *Raft) AppendEntries(args *AppendEntriesArgs, reply *AppendEntriesReply
 	}
 
 	if len(args.Entries) == 0 {
-		// TODO(heng): dealing with heatbeat here
+		// Handle heartbeat
```

Comment:
We should check `IsLogConsecutive` after the heartbeat.

Reply:
Done